### PR TITLE
Updating ETL config to incorporate unify logging.

### PR DESCRIPTION
**Pull request by ShridharSattur**
Updating ETL util to incorporate unify logging.

### CODE REVIEW

File: wherehows-backend/conf/etl_logback.xml [hunk 1]
```diff
@@ -25,13 +25,13 @@
     </appender>
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <!--See also http://logback.qos.ch/manual/appenders.html#RollingFileAppender-->
-        <File>${LOG_DIR}/${CONTEXT:-default}.log</File>
+        <File>${LOG_DIR}/${CONTEXT:-default}/${execId}.log</File>
```
Comment (mars-lan) [inline wherehows-backend/conf/etl_logback.xml]: ${execId}/log

Comment (ShridharSattur) [reply wherehows-backend/conf/etl_logback.xml]: Updated it.

File: wherehows-backend/app/actors/ConfigUtil.java [hunk 2]
```diff
@@ -72,6 +73,7 @@ static ProcessBuilder buildProcess(String etlJobName, long whEtlExecId, String c
         .add("-cp")
         .add(classPath)
         .add("-Dconfig=" + configFile)
+        .add("-DexecId=" + whEtlExecId)
```
Comment (mars-lan) [inline wherehows-backend/app/actors/ConfigUtil.java]: This is probably not needed any more?

File: wherehows-backend/app/actors/ConfigUtil.java [hunk 3]
```diff
@@ -63,7 +64,7 @@ static ProcessBuilder buildProcess(String etlJobName, long whEtlExecId, String c
       Properties etlJobProperties) {
     String classPath = System.getProperty("java.class.path");
     String outDir = etlJobProperties.getProperty(Constant.WH_APP_FOLDER_KEY, WH_APPLICATION_DEFAULT_DIRECTORY);
-    String configFile = outDir + "/exec/" + whEtlExecId + ".properties";
+    String configFile =  outDir + "/" + whEtlExecId + ".properties";
```
Comment (mars-lan) [inline wherehows-backend/app/actors/ConfigUtil.java]: Drop the extra space?

File: wherehows-backend/app/actors/EtlJobActor.java [hunk 4]
```diff
@@ -51,7 +51,7 @@
 
   @Override
   public void onReceive(Object message) throws Exception {
-    final String configDir = ETL_TEMP_DIR + "/exec";
+    String configDir = ETL_TEMP_DIR;
```
Comment (mars-lan) [inline wherehows-backend/app/actors/EtlJobActor.java]: We can just set configDir = ETL_TTEMP_DIR + "/" + msg.getEtlJobName() + "/" + msg.getWhEtlExecId() here instead?

File: wherehows-backend/app/actors/ConfigUtil.java (hunk 3 above)
Comment (ShridharSattur) [reply wherehows-backend/app/actors/ConfigUtil.java]: Reformatted the code now which removed extra space.

File: wherehows-backend/app/actors/ConfigUtil.java (hunk 2 above)
Comment (ShridharSattur) [reply wherehows-backend/app/actors/ConfigUtil.java]: Removed it.

File: wherehows-backend/app/actors/EtlJobActor.java (hunk 4 above)
Comment (ShridharSattur) [reply wherehows-backend/app/actors/EtlJobActor.java]: We can't set the directory structure here as we are constructing the etljob properties(msg object) only after this line.

**Pull request merged**